webpack/assets/javascripts/react_app/components/factCharts/FactChart.test.js

```diff
@@ -0,0 +1,134 @@
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import React from 'react';
+import FactChart from '.';
+import Store from '../../redux';
+
+import { getChartData, showModal, closeModal } from './FactChartActions';
+
+import { ajaxRequestAction } from '../../redux/actions/common/index';
+
+import reducer from './FactChartReducer';
+import * as types from './FactChartConstants';
+
+import {
+  initialState,
+  modalOpenState,
+  modalSuccessState,
+  modalLoadingState,
+  modalErrorState,
+  chartDataValues,
+} from './FactChart.fixtures';
+
+describe('factCharts', () => {
```

Comment:
there is a helper function for making snapshots - `testComponentSnapshotsWithFixtures` from testsHelper

Reply:
only the reducer and action tests are using fixtures so its not needed